kernel_crawler/bottlerocket.py

```diff
@@ -90,52 +90,74 @@ def get_package_tree(self, version=''):
             self.checkout_version(v)
 
             # Find supported kernels dynamically
-            supported_kernel_specs = self.match_file("kernel-.*.spec", False)
+            supported_kernel_specs = self.match_file("kernel-.*.spec", True)
```

Comment:
Fetch the full path to kernel specs, so that we can use their `dirname` as `working dir` when looking up for config file and supported flavors.
This is a fix.